packaging/debian-unstable/patches/pb.v1-canonical-path.patch

```diff
@@ -0,0 +1,20 @@
+Description: switch to canonical path for gopkg.in/cheggaaa/pb.v1
```

Comment:
Why are we not using the canonical path?

Reply:
Again, this is fixed in master.